pkg/authorize/jwt/client_authorizer.go

```diff
@@ -0,0 +1,97 @@
+package jwt
+
+import (
+	"crypto"
+	"encoding/base64"
+	"encoding/json"
+	"strings"
+
+	"github.com/openshift/telemeter/pkg/authorize"
+	"gopkg.in/square/go-jose.v2/jwt"
+)
+
+// NewClientAuthorizer authenticates tokens as JWT tokens produced by JWTTokenGenerator
```

Comment:
`NewClientAuthorizer` does not actually authenticate anything, right? It is only a constructor. We should follow up and fix these docs since they are a little misleading. We could write this for the godoc for the returned type.

Also, since this type is `...Authorizer` we should explicitly document what authorization check is being performed. Is the policy for authorization simply verifying the issuer and signer, and anyone matching this rule is fully authorized for all access? Otherwise we are only authenticating.

Reply:
ack, yes, you have a good point, I shifted some godocs around in order to not loose any language from old code. In the next pass I definitely want to complete godoc :+1: